### PR TITLE
Fixed deprecated Links, typos and Grammatical errors in corporate-dir…

### PR DESCRIPTION
**Pull request by ArmanChand**
…ectory-integration.rst

### CODE REVIEW

File: source/about/corporate-directory-integration.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ Security features for authentication
 
 A core set of features is available with all authentication options to help increase security:
 
-- Ability to `set session length <https://docs.mattermost.com/administration/config-settings.html#id33>`__ to define how long a user can use Mattermost before needing to re-enter credentials.
+- Ability to `set session length <https://docs.mattermost.com/configure/configuration-settings.html>`__ to define how long a user can use Mattermost before needing to re-enter credentials.
```
Comment (cwarnermm) [inline source/about/corporate-directory-integration.rst:22]:
```suggestion
- Ability to `set session length <https://docs.mattermost.com/configure/configuration-settings.html#session-lengths>`__ to define how long a user can use Mattermost before needing to re-enter credentials.
```

Comment (cwarnermm) [reply source/about/corporate-directory-integration.rst:22]: I've provided the updated URL.

**Review by cwarnermm — APPROVED** [review]
Thank you for your docs contribution!

**Comment by ArmanChand** [comment]
@cwarnermm ,kindly merge this PR

**Review by justinegeffen — APPROVED** [review]
Thank you!

**Pull request merged**